support-e2e/tests/test/checkout.ts

```diff
@@ -27,6 +27,7 @@ const setUserDetailsForProduct = async (
 	switch (product) {
 		case 'SupporterPlus':
 		case 'GuardianLight':
```

Comment:
We can remove the old name from the test can't we?

Reply:
Further down the product is defined in the Url of the checkout (`checkout.ts` line62). Think this has to match the Zuora product Catalog (`product not found`) so needs to stay to preserve compatibility until this new product is available in the Zuora Product catalog. This chnage causes the checkout/guardian-light smoke tests to fail